scripts/obtain/dotnet-install.sh

```diff
@@ -653,7 +717,7 @@ do
             echo "Options:"
             echo "  -c,--channel <CHANNEL>         Download from the CHANNEL specified (default: $channel)."
             echo "      -Channel"
-            echo "  -v,--version <VERSION>         Use specific version, ``latest`` or ``lkg``. Defaults to ``latest``."
+            echo "  -v,--version <VERSION>         Use specific version, ``latest``. Defaults to ``latest``."
```

Comment:
Also, this was already broken before, but the backtick escaping is wrong. It should be "\\\`", not "\`\`":

    echo "  -v,--version <VERSION>         Use specific version, \`latest\`. Defaults to \`latest\`."